man/Makefile.am

```diff
@@ -57,6 +56,7 @@ man_nopam = \
 
 if WITH_SU
 man_MANS += man1/su.1
+man_nopam += man5/suauth.5
 endif
```

Comment:
This doesn't seem to remove the page if WITH_SU but !USE_PAM, right?

Reply:
That's the point. If you have su without PAM, /etc/suauth is parsed.

Reply:
Oops, I probably meant WITH_SU and USE_PAM.